src/rdpBuilder.js

```diff
@@ -0,0 +1,177 @@
+
+const BASE_CONFIG_FILE = 'session bpp:i:32\n' +
+                          'winposstr:s:0,3,0,0,800,600\n' +
+                          'compression:i:1\n' +
+                          'keyboardhook:i:2\n' +
+                          'audiocapturemode:i:0\n' +
+                          'videoplaybackmode:i:1\n' +
+                          'connection type:i:2\n' +
+                          'displayconnectionbar:i:1\n' +
+                          'disable wallpaper:i:1\n' +
+                          'allow font smoothing:i:0\n' +
+                          'allow desktop composition:i:0\n' +
+                          'disable full window drag:i:1\n' +
+                          'disable menu anims:i:1\n' +
+                          'disable themes:i:0\n' +
+                          'disable cursor setting:i:0\n' +
+                          'bitmapcachepersistenable:i:1\n' +
+                          'audiomode:i:0\n' +
+                          'redirectcomports:i:0\n' +
+                          'redirectposdevices:i:0\n' +
+                          'redirectdirectx:i:1\n' +
+                          'autoreconnection enabled:i:1\n' +
+                          'prompt for credentials:i:1\n' +
+                          'negotiate security layer:i:1\n' +
+                          'remoteapplicationmode:i:0\n' +
+                          'alternate shell:s:\n' +
+                          'shell working directory:s:\n' +
+                          'gatewayhostname:s:\n' +
+                          'gatewayusagemethod:i:4\n' +
+                          'gatewaycredentialssource:i:4\n' +
+                          'gatewayprofileusagemethod:i:0\n' +
+                          'promptcredentialonce:i:1\n' +
+                          'use redirection server name:i:0'
+export default class RDPBuilder {
+  constructor (vmName, username) {
+    this.address = vmName
+    this.guestID = ''
+    this.fullScreen = true
+    this.fullScreenTitle
+    this.width = 640
+    this.height = 480
+    this.authenticationLevel = 2
+    this.useLocalDrives
+    this.redirectPrinters = false
+    this.redirectClipboard = true
+    this.redirectSmartCards = false
+    this.username = username
+  }
+
+  getUsername () {
```

Comment:
I'm sorry but I really don't see the value in this bunch of trivial getters & setters. If some special logic would indeed be required, it would be better to implement the getters & setters using the new ES6 syntax like this http://stackoverflow.com/questions/28222276/what-are-getters-and-setters-for-in-ecmascript-6-classes

So let's stick to using the good old dot `.` operator :-)